pkg/debug/scripts/collect-info.sh

```diff
@@ -45,6 +59,57 @@ while getopts "vh" o; do
     esac
 done
 
+sort_cat_jq()
+{
+    # Sort and extract a filename
+    sort -n | awk '{print $2}' | \
+    # Decompress if needed and output
+    xargs --no-run-if-empty zcat -f | \
+    # Concat timestamp and make it human readable. The whole complexity lies
+    # in the JQ `strftime` which does not support milli/nano seconds, and for
+    # me that means converting nanos to a fraction of a float number. If nanos
+    # field does not have exactly 9 digits it should be prepended with 0, look
+    # on this example: nanos == 99, so the fraction should be ".000000099" and
+    # not ".99", that's why "nanos + 1e9 | tostring | .[1:]", which does this
+    # conversion. Also:
+    # "-R (. as $line | try fromjson)" means ignore a line if is not a JSON,
+    # "(nanos // 0)" means return 0 if nanos is null
+    jq -R '(. as $line | try fromjson) | .timestamp_str.nanos = ((.timestamp.nanos // 0) + 1e9 | tostring | .[1:]) | .timestamp_str.human = (.timestamp.seconds | strftime("%B %d %Y %I:%M:%S")) | .timestamp_str = "\(.timestamp_str.human).\(.timestamp_str.nanos)"'
```

Comment:
Could you please put to the comment an example of the transformed JSON entity? Something like:
```json
{
  "message": "Some log message",
  "timestamp": {
    "seconds": 1672444800,
    "nanos": 123456789
  },
  "timestamp_str": {
    "nanos": ".123456789",
    "human": "January 01 2023 12:00:00"
  }
}
```
It's a bit confusing with all the formats of the timestamps we have... The example will help to understand and to implement parsing scripts.

Reply:
Ii's a bit different, this is the output:
```
  "timestamp": {
    "seconds": 1677023830,
    "nanos": 399834057
  },
  "timestamp_str": "February 21 2023 11:57:10.399834057"
```

Yes, I can describe why I create a new json entry and how it looks like.